regulations/static/regulations/css/less/mixins.less

```diff
@@ -5,28 +5,42 @@ mixins.less contains any mixins to be used throughout the LESS files
 */
 
 
-.font-regular {
-    font-family: "Avenir Next", Arial, sans-serif;
-    font-weight: normal;
+
+.sans-font-light {
+    font-family: "Source Sans Pro", Arial, sans-serif;
+    font-weight: 300;
     font-style: normal;
 }
 
-.font-bold {
-    font-family: "Avenir Next Demi", Arial, sans-serif;
-    font-weight: 600;
+.sans-font-regular {
+    font-family: "Source Sans Pro", Arial, sans-serif;
+    font-weight: 400;
+    font-style: normal;
 }
 
-.font-italic {
-    font-family: "Avenir Next Demi Italic", Arial, sans-serif;
+.sans-font-bold {
+    font-family: "Source Sans Pro", Arial, sans-serif;
     font-weight: 600;
+    font-style: normal;
+}
+
+.serif-font-italic {
+    font-family: "Merriweather", Georgia, serif;
+    font-weight: 400;
     font-style: italic;
 }
 
+.serif-font-regular {
+  font-family: "Merriweather", Georgia, serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
 
 /* Drawer font styles */
 
 .drawer-header-font {
-    &:extend(.font-bold);
+    &:extend(.sans-font-bold);
```

Comment:
The `&:extend()` usage here needed targeted conversion for scss. Can you check to see if just delcaring `.sans-font-bold` works the same way?